pkg/urbit/vere/io/http.c

```diff
@@ -1738,7 +1737,9 @@ _http_serv_start_all(u3_httd* htd_u)
 
     //  XX remove [sen]
     //
-    u3_noun wir = u3nt(u3i_string("http-server"), u3k(u3A->sen), u3_nul);
+    u3_noun wir = u3nt(u3i_string("http-server"),
+                       u3dc("scot", c3__uv, htd_u->sev_l),
```

Comment:
Aren't these various `scot` calls rather expensive?

Reply:
Yeah, they are. Here and above, we should really call them once and store them in the driver/server/connection/request structs as strings. I'll note that for later.